backend/src/main/game/cards/test_cards.py

```diff
@@ -0,0 +1,29 @@
+from backend.src.main.game.cards.card import Card
+from backend.src.main.game.cards.cards import Cards
+
+
+def test_cards_starts_empty():
+    assert Cards().cards.__len__() == 0
+
+
+def test_add_card_to_card_deck():
+    cards = Cards()
+    card_one = Card("card1")
+    cards.append(card_one)
+    assert cards.cards.__len__() == 1
+    assert cards.cards.__getitem__(0) == card_one
```

Comment:
change to `cards.cards[0]` or add a `get(index: int)` function to your `Cards` class if you want to hide the choice of data structure